app/src/main/java/com/sdp13epfl2021/projmag/activities/CVCreationActivity.kt

```diff
@@ -35,7 +37,9 @@ class CVCreationActivity : AppCompatActivity() {
 
     private fun onFinish() {
         val cv = buildCV()
-        /* DO SOMETHING WITH IT */
+        val utils = Utils.getInstance(this)
+        val userDataFirebase = utils.userDataDatabase
+        userDataFirebase.pushCv(cv, { Toast.makeText(this, "Success!", Toast.LENGTH_LONG)}, {Toast.makeText(this, "Failure!", Toast.LENGTH_LONG)})
```

Comment:
The Toast are now shown, and maybe using resources strings would better
```suggestion
        userDataFirebase.pushCv(cv, {
            Toast.makeText(this, "Success!", Toast.LENGTH_LONG).show()
        }, {
            Toast.makeText(this, "Failure!", Toast.LENGTH_LONG).show()
        })
```